components/pictures/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {Fragment} from 'react';
+import axios from 'axios';
+import useSWR, {Key} from "swr";
+import {
+  ImageList,
+  ImageListItem,
+  Skeleton
+} from "@mui/material";
+import {Box} from '@mui/system';
+import PictureTile, {ApodType} from "../pictureTile";
+
+import useMediaQuery from '@mui/material/useMediaQuery';
+
+export const fetcher = (url: string) => axios.get(url).then(res => res.data)
+
+
+function Pictures() {
+  const apiURL: Key = `https://api.nasa.gov/planetary/apod?start_date=${"2022-01-12"}&end_date=${"2022-01-26"}&thumbs=true&api_key=${process.env.NEXT_PUBLIC_NASA_API_KEY}`;
```

Comment:
Why are you evaluating literals in `${"2022-01-12"}`? :thinking: , I think you can just hardcode them if you are not evaluating anything inside